docs/ff-concepts/adding-customization/common-examples.md

```diff
@@ -0,0 +1,284 @@
+---
+slug: /concepts/custom-code/common-examples
+title: Common Examples
+description: Learn about the common custom code examples and use it directly in your project.
+tags: [Custom Actions, Custom Code]
+sidebar_position: 6
+keywords: [FlutterFlow, Custom Actions, Customizations, Flutter, Dart, Pub.dev, Examples]
+---
+
+# Common Code Examples
+
+The custom code feature in FlutterFlow allows you to extend functionality by accessing generated classes and modifying global variables like App States and FlutterFlow themes. This guide covers common scenarios where you can leverage custom code to enhance your project by working directly with data models and other resources within your code.
+
+### Accessing FlutterFlow Generated Classes
+
+Most custom FlutterFlow classes are prefixed with `FF<ClassName>` or `FlutterFlow<ClassName>`. If you need to access these classes in your custom code, simply type "FF" or "FlutterFlow" in the code editor to locate them easily.
```

Comment:
Would recommend specifying what classes means here. In this case it sounds like widgets we use under the hood, as opposed to components which are also FlutterFlow Generated Classes. 

Suggest putting a one liner like "In the generated code, FlutterFlow uses a combination of widgets from the Flutter framework, popular pub.dev packages and custom widgets created by the FlutterFlow team. You may want to reference widgets created by the FlutterFlow team in your own generated code, such as FFDropdown or ..... these widgets are usually prefixed...."

Reply:
thats why it is helpful to explain what a class means in this context :)

Reply:
Makes sense! Done!